### PR TITLE
Fixed wrong or missing phpdoc

### PR DESCRIPTION
**Pull request by core23**
Fixes some phpdoc that was missing, wrong or inherited.

### CODE REVIEW

File: Controller/GroupController.php [hunk 1]
```diff
@@ -43,6 +47,10 @@ public function listAction()
 
     /**
      * Show one group
+     *
+     * @param $groupName
```
Comment (OskarStark) [inline Controller/GroupController.php]: type string

File: Controller/GroupController.php [hunk 2]
```diff
@@ -55,12 +63,17 @@ public function showAction($groupName)
 
     /**
      * Edit one group, show the edit form
+     *
+     * @param Request $request
+     * @param         $groupName
```
Comment (OskarStark) [inline Controller/GroupController.php]: type string

File: Controller/GroupController.php [hunk 3]
```diff
@@ -147,6 +164,11 @@ public function newAction(Request $request)
 
     /**
      * Delete one group
+     *
+     * @param Request $request
+     * @param         $groupName
```
Comment (OskarStark) [inline Controller/GroupController.php]: type string

File: Controller/RegistrationController.php [hunk 4]
```diff
@@ -109,6 +118,11 @@ public function checkEmailAction()
 
     /**
      * Receive the confirmation token from user email provider, login the user
+     *
+     * @param Request $request
+     * @param         $token
```
Comment (OskarStark) [inline Controller/RegistrationController.php]: string

File: Controller/ResettingController.php [hunk 5]
```diff
@@ -125,6 +135,11 @@ public function checkEmailAction(Request $request)
 
     /**
      * Reset user password
+     *
+     * @param Request $request
+     * @param         $token
```
Comment (OskarStark) [inline Controller/ResettingController.php]: string

File: DependencyInjection/FOSUserExtension.php [hunk 6]
```diff
@@ -194,6 +219,12 @@ private function loadResetting(array $config, ContainerBuilder $container, XmlFi
         ));
     }
 
+    /**
+     * @param array            $config
+     * @param ContainerBuilder $container
+     * @param XmlFileLoader    $loader
+     * @param                  $dbDriver
```
Comment (OskarStark) [inline DependencyInjection/FOSUserExtension.php]: string

File: Doctrine/GroupManager.php [hunk 7]
```diff
@@ -12,15 +12,33 @@
...
+    /**
+     * @var ObjectRepository
+     */
     protected $repository;
 
+    /**
+     * GroupManager constructor.
+     *
+     * @param ObjectManager $om
+     * @param               $class
```
Comment (OskarStark) [inline Doctrine/GroupManager.php]: string

File: Doctrine/UserManager.php [hunk 8]
```diff
@@ -19,8 +19,19 @@
...
+     */
     protected $objectManager;
+
+    /**
+     * @var string
+     */
     protected $class;
+
+    /**
+     * @var \Doctrine\Common\Persistence\ObjectRepository
```
Comment (OskarStark) [inline Doctrine/UserManager.php]: please use use statement here

File: EventListener/AuthenticationListener.php [hunk 9]
```diff
@@ -24,12 +24,21 @@ class AuthenticationListener implements EventSubscriberInterface
     private $loginManager;
     private $firewallName;
 
+    /**
+     * AuthenticationListener constructor.
+     *
+     * @param LoginManagerInterface $loginManager
+     * @param                       $firewallName
```
Comment (OskarStark) [inline EventListener/AuthenticationListener.php]: string

File: EventListener/FlashListener.php [hunk 10]
```diff
@@ -65,6 +78,12 @@ public function addSuccessFlash(Event $event, $eventName = null)
         $this->session->getFlashBag()->add('success', $this->trans(self::$successMessages[$eventName]));
     }
 
+    /**
+     * @param       $message
```
Comment (OskarStark) [inline EventListener/FlashListener.php]: string

File: EventListener/ResettingListener.php [hunk 11]
```diff
@@ -23,12 +23,21 @@ class ResettingListener implements EventSubscriberInterface
     private $router;
     private $tokenTtl;
 
+    /**
+     * ResettingListener constructor.
+     *
+     * @param UrlGeneratorInterface $router
+     * @param                       $tokenTtl
```
Comment (OskarStark) [inline EventListener/ResettingListener.php]: not sure about the type, can you investigate? string or int

File: Form/Factory/FormFactory.php [hunk 12]
```diff
@@ -15,11 +15,34 @@
...
+    /**
+     * @var array
+     */
     private $validationGroups;
 
+    /**
+     * FormFactory constructor.
+     *
+     * @param FormFactoryInterface $formFactory
+     * @param                      $name
```
Comment (OskarStark) [inline Form/Factory/FormFactory.php]: string

File: Form/Factory/FormFactory.php [hunk 13]
```diff
@@ -15,11 +15,34 @@
...
+     * @param                      $type
```
Comment (OskarStark) [inline Form/Factory/FormFactory.php]: string

File: Model/Group.php [hunk 14]
```diff
@@ -16,20 +16,35 @@
...
+
+    /**
+     * @var array
+     */
     protected $roles;
 
+    /**
+     * Group constructor.
+     *
+     * @param       $name
```
Comment (OskarStark) [inline Model/Group.php]: string

File: Model/UserInterface.php [hunk 15]
```diff
@@ -19,7 +19,14 @@
  */
 interface UserInterface extends AdvancedUserInterface, \Serializable
 {
+    /**
+     *
```
Comment (OskarStark) [inline Model/UserInterface.php]: ?

File: Model/UserInterface.php [hunk 16]
```diff
@@ -19,7 +19,14 @@
...
+     */
     const ROLE_DEFAULT = 'ROLE_USER';
+
+    /**
+     *
```
Comment (OskarStark) [inline Model/UserInterface.php]: ?

File: Security/LoginManager.php [hunk 17]
```diff
@@ -36,6 +36,14 @@ class LoginManager implements LoginManagerInterface
     private $sessionStrategy;
     private $container;
 
+    /**
+     * LoginManager constructor.
+     *
+     * @param                                        $tokenStorage
```
Comment (OskarStark) [inline Security/LoginManager.php]: `TokenStorageInterface|SecurityContextInterface`

File: Tests/Doctrine/UserManagerTest.php [hunk 18]
```diff
@@ -7,6 +7,9 @@
 
 class UserManagerTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     *
```
Comment (OskarStark) [inline Tests/Doctrine/UserManagerTest.php]: ?

File: Tests/EventListener/AuthenticationListenerTest.php [hunk 19]
```diff
@@ -8,6 +8,9 @@
 
 class AuthenticationListenerTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     *
```
Comment (OskarStark) [inline Tests/EventListener/AuthenticationListenerTest.php]: ?

File: Tests/Security/LoginManagerTest.php [hunk 20]
```diff
@@ -57,6 +57,13 @@ public function testLogInUserWithRememberMeAndWithoutRequestStack()
         $loginManager->logInUser('main', $this->mockUser(), $response);
     }
 
+    /**
+     * @param               $firewallName
```
Comment (OskarStark) [inline Tests/Security/LoginManagerTest.php]: string

File: Tests/Routing/RoutingTest.php [hunk 21]
```diff
@@ -10,6 +10,10 @@ class RoutingTest extends \PHPUnit_Framework_TestCase
 {
     /**
      * @dataProvider loadRoutingProvider
+     *
+     * @param       $routeName
```
Comment (OskarStark) [inline Tests/Routing/RoutingTest.php]: string

File: Tests/Routing/RoutingTest.php [hunk 22]
```diff
@@ -10,6 +10,10 @@ class RoutingTest extends \PHPUnit_Framework_TestCase
...
+     * @param       $path
```
Comment (OskarStark) [inline Tests/Routing/RoutingTest.php]: string

**Comment by core23** [comment]
Fixed

**Comment by OskarStark** [comment]
@XWB can you please merge this?

**Comment by XWB** [comment]
Thanks.

**Pull request merged**